lib/compilers/kotlin.js

```diff
@@ -54,9 +54,6 @@ export class KotlinCompiler extends JavaCompiler {
             option !== '-script' && option !== '-progressive' && !option.startsWith('-Xjavac'));
 
         const oneArgForbiddenList = new Set([
-            // -d directory
-            // Destination for generated class files
-            '-d',
```

Comment:
Unlucky workaround to allow handleInterpreting to pass down '-d' when executing code. Should not really matter as we run execution in isolated environments in prod.